static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -102,12 +102,20 @@ const defaultPageCheck = (page: Object): boolean =>
 
 const shouldShowReaderRevenue = (
     showToContributorsAndSupporters: boolean = false
-): boolean =>
-    (userShouldSeeReaderRevenue() || showToContributorsAndSupporters) &&
-    !config.page.keywordIds.includes(
-        'guardian-masterclasses/guardian-masterclasses'
-    ) &&
-    !config.page.shouldHideReaderRevenue;
+): boolean => {
+    const isMasterclassesPage =
+        config.page &&
+        config.page.keywordIds &&
+        config.page.keywordIds.includes(
+            'guardian-masterclasses/guardian-masterclasses'
+        );
```

Comment:
For the future: you could achieve the same with `config.get('page.keywordIds', []).includes()`. [Example specs](https://github.com/guardian/frontend/blob/master/static/src/javascripts/lib/config.spec.js#L54).